commit 7563e0bbb54fbac54e77f3a62d9761a70f3a559c
Author: Colin Guthrie <colin@mageia.org>
Date:   Sat Oct 1 12:03:44 2011 +0100

    libpulse: Always return a three part version number in API calls.
    
    For both the headers and the library we should provide clean, three part
    strings as this has been what we've previously done in the past
    and some external systems apparently rely on this format. While it's not
    something we've officially commented on before, there is no real advantage
    to us to change it so let's not try to tidy things up too much
    considering some third party apps (e.g. Skype) seem to dislike a two
    part version string.

diff --git a/src/pulse/context.c b/src/pulse/context.c
index 25d04a1..af144aa 100644
--- a/src/pulse/context.c
+++ b/src/pulse/context.c
@@ -1281,7 +1281,7 @@ pa_operation* pa_context_set_name(pa_context *c, const char *name, pa_context_su
 }
 
 const char* pa_get_library_version(void) {
-    return PACKAGE_VERSION;
+    return pa_get_headers_version();
 }
 
 const char* pa_context_get_server(pa_context *c) {
diff --git a/src/pulse/version.h.in b/src/pulse/version.h.in
index 7e00c5e..1be4c75 100644
--- a/src/pulse/version.h.in
+++ b/src/pulse/version.h.in
@@ -35,7 +35,7 @@ PA_C_DECL_BEGIN
 /** Return the version of the header files. Keep in mind that this is
 a macro and not a function, so it is impossible to get the pointer of
 it. */
-#define pa_get_headers_version() ("@PACKAGE_VERSION@")
+#define pa_get_headers_version() ("@PA_MAJOR@.@PA_MINOR@.0")
 
 /** Return the version of the library the current application is
  * linked to. */
